libmscore/rendermidi.cpp

```diff
@@ -352,11 +352,10 @@ static void collectNote(EventMap* events, int channel, const Note* note, int vel
                           /  .                   midi pitch is 12/16384 semitones
                          A....
                        tickDelta   */
-                  for (int i = lastPointTick; i <= nextPointTick; i += 16) {
+                  for (int i = lastPointTick; i <= nextPointTick; i += 1) {
```

Comment:
Is `i++` more/less readable than `i += 1` ?

Reply:
++ is probably correct, I'll change this (considering I have to update the midi tests anyway)